pkg/components/monitoring/component.go

```diff
@@ -37,6 +37,11 @@ var MonitoringComponent = Component{
 			{
 				IncludeAll: []string{"cluster-monitoring-operator"},
 			},
+			{Suite: "Cluster_Observability"},
+			{Suite: "Symptom Detection"},
+			{Suite: "cluster monitoring related upgrade check"},
+			{Suite: "servicemonitor"},
+			{Suite: "serviceXXXitor"},
```

Comment:
no
```
{Suite: "Symptom Detection"},
{Suite: "servicemonitor"},
{Suite: "serviceXXXitor"},
```
missed cucushift suites
`Install and configuration related scenarios`, see from https://github.com/openshift/cucushift/blob/master/monitoring/ci.feature#L1
`alerts browser`, see from https://github.com/openshift/cucushift/blob/master/web/monitoring/alerts-browser.feature#L1
`query browser`, see from https://github.com/openshift/cucushift/blob/master/web/monitoring/query-browser.feature#L1

also `Monitoring dashboards related features` for monitoring cypress UI cases
https://github.com/openshift/openshift-tests-private/blob/master/frontend/tests/monitoring/dashboards.cy.ts#L2